observer.go

```diff
@@ -231,9 +231,13 @@ func (o *Observer) refresh(ch chan<- prometheus.Metric) (map[string]string, erro
 			infoKeys = append(infoKeys, keys...)
 		}
 	}
+	// append infoKey "build" - this is removed from WatcherLatencies to avoid forced watcher sequence during refresh
+	infoKeys = append(infoKeys, "build")
```

Comment:
```suggestion
	// append infoKey "build" - this is removed from WatcherLatencies to avoid forced watcher sequence during refresh
	infoKeys = append(infoKeys, "build")
```
and line 228 can be changed from
```
	var infoKeys []string
```

```
	infoKeys := []string{ikBuild}
```